compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIETileAndFuse.cpp

```diff
@@ -108,6 +111,15 @@ void AMDAIETileAndFusePass::runOnOperation() {
     // the skip ops list which currently contains linalg.copy and tensor.unpack.
     if (op.getLoopIteratorTypes().empty() || consumerToSkip(op))
       return WalkResult::advance();
+
+    // For matmul + elementwise dispatch, we use flag `tileElementwise` to
+    // indicate whether we want to tile the elementwise op. If flag
+    // `tileElementwise == false`, and the linalg op is an elementwise op, it
+    // will advance to find the next target op for tiling.
+    auto linalgOp = dyn_cast_or_null<linalg::LinalgOp>(op.getOperation());
```

Comment:
Isnt tile elementwise always false. In which case we are just avoiding elementwise op tiling always? If so, I'd just make that the case by default and avoid all the complexity with the flag. I see why you have it. Its OK to leave it in there too.

Reply:
In the current setting, for the first level we'd like to tile both matmul and elementwise ops, in which case `tileElementwise == true`, and when it finds the elementwise op the walk function will be interrupted.